src/translators/crate.py

```diff
@@ -701,6 +701,75 @@ def query(self,
             result.extend(entities)
         return result
 
+    def query_ids(self,
+              entity_type=None,
+              from_date=None,
+              to_date=None,
+              limit=10000,
+              offset=0,
+              fiware_service=None,
+              fiware_servicepath=None):
+        if limit == 0:
+            return []
+
+        where_clause = self._get_where_clause(None,
+                                              from_date,
+                                              to_date,
+                                              fiware_servicepath,
+                                              None)
+
+        if entity_type:
+            table_names = [self._et2tn(entity_type, fiware_service)]
+        else:
+            table_names = self._get_et_table_names(fiware_service)
+
+        if fiware_service is None:
+            for tn in table_names:
+                if "." in tn:
+                    table_names.remove(tmp)
+
+        limit = min(10000, limit)
+        offset = max(0, offset)
+        len_tn = 0
+        result = []
+        stmt = ''
+        for tn in table_names:
+            len_tn += 1
+            if len_tn != len(table_names):
+                stmt += "select entity_id, entity_type, max(time_index) " \
+                           "from {tn} {where_clause}" \
+                           "group by entity_id, entity_type " \
+                           "union all ".format(
+                              tn=tn,
+                              where_clause=where_clause
+                           )
+            else:
+                stmt += "select entity_id, entity_type, max(time_index) " \
+                           "from {tn} {where_clause}" \
+                           "group by entity_id, entity_type ".format(
+                              tn=tn,
+                              where_clause=where_clause
+                           )
+
+        op = stmt + "limit {limit} offset {offset}".format(
```

Comment:
as suggested, it may be a good idea ordering them by time index

Reply:
yep, see also my comment above about failing test case.

Reply:
I have added code for order by time-index in ascending order as used in Orion.

Reply:
good stuff, thanks a stack!